### ISSUE TITLE
Load remote data from a backend ?

### ISSUE THREAD

**Issue by sahanDissanayake** [documentcloud/visualsearch#147]
Hi 

Is there a way to load the results from a backend ? and then show them to the user ? Like I have a list of names which is about 1300 and then I need a way to autocomplete like Google, Should I use this plugin or just go for something else ?

Thanks everyone

**Comment by kaystrobach** [comment]
yes it's possible http://stackoverflow.com/questions/18381933/implementing-visualsearch-js-using-jquery-ajax-rest-api-and-neo4j-graphdb
